core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2375,4 +2376,7 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    @LogMessage(level = WARN)
    @Message(value = "Failed to register metrics with id %s. Reason: %s", id = 970)
    void metricRegistrationFailed(String id, String reason);
+
+   @Message(value = "Simple caches do not support queries. Use standard caches for querying.", id = 971)
```

Comment:
This message doesn't seem appropriate for the state we're trying to describe :slightly_smiling_face:

Reply:
Let me fire the AI bot that wrote it :rofl: